core/src/main/java/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java

```diff
@@ -173,12 +174,33 @@ private Answer execute(final SetupKeyStoreCommand cmd) {
         return new SetupKeystoreAnswer(result.getDetails());
     }
 
+    private void scpFileToIdentifyPatching(String routerIp, String path, String filename, String content) throws InterruptedException {
+        String errMsg = "Failed to scp file: %s to system VM";
+        for (int retries = 15; retries > 0; retries--) {
+            try {
+                _vrDeployer.createFileInVR(routerIp, path, filename, content);
+            } catch (Exception e) {
+                errMsg += ", retrying";
+                s_logger.error(String.format(errMsg, filename), e);
+                Thread.sleep(2000);
+            }
+        }
+    }
     private Answer execute(final SetupCertificateCommand cmd) {
-        final String args = String.format("/usr/local/cloud/systemvm/conf/agent.properties " +
+        String routerName = cmd.getAccessDetail(NetworkElementCommand.ROUTER_NAME);
```

Comment:
I see this isn't really router name, but name of the systemvm right?

Reply:
yes, this is the systemvm name, but historically, we've used the variable  'ROUTE_NAME' to store the name of the systemvm